app_dart/lib/src/request_handlers/refresh_github_commits.dart

```diff
@@ -75,186 +72,31 @@ class RefreshGithubCommits extends ApiRequestHandler<Body> {
         continue;
       }
 
-      final List<Commit> newCommits = await _getNewCommits(commits, datastore, branch);
-
-      if (newCommits.isEmpty) {
-        // Nothing to do.
-        continue;
-      }
-      log.debug('Found ${newCommits.length} new commits for branch $branch on GitHub');
-
-      // Save [Commit] to BigQuery and create [Task] in Datastore.
-      await _saveData(newCommits, datastore);
+      final List<Commit> recentCommits = await _getRecentCommits(commits, datastore, branch);
```

Comment:
Move this logic to scheduler as well to save both time and space (processing `RepositoryCommit` directly)

Reply:
I left this code here as in the follow up PR I want to remove it completely (move to a single commit from a webhook). SG?